### PR TITLE
08.Docker

### PR DESCRIPTION
**Pull request by anton-bartoshevich**

### CODE REVIEW

File: Anton_Bartoshevich/08.Docker.Docker-compose/docker-image.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name: Docker Image CI
+
+on:
+  push:
```
Comment (pluhin) [inline Anton_Bartoshevich/08.Docker.Docker-compose/docker-image.yml:4]: according to the task, you need to use tag as initiator of deployment

File: Anton_Bartoshevich/08.Docker.Docker-compose/docker-image.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Build the Docker image
+      run: docker build . --file Dockerfile --tag my-image-name:$(date +%s)
```
Comment (pluhin) [inline Anton_Bartoshevich/08.Docker.Docker-compose/docker-image.yml:18]: this is only build, there is no push in your github action

**Review by pluhin — CHANGES_REQUESTED** [review]

File: Anton_Bartoshevich/08.Docker.Docker-compose/docker-image.yml (hunk 2 above)
Comment (anton-bartoshevich) [reply Anton_Bartoshevich/08.Docker.Docker-compose/docker-image.yml]: Sorry, the .yml file is wrong, I uploaded the correct .yml file.

File: Anton_Bartoshevich/08.Docker.Docker-compose/docker-image.yml (hunk 1 above)
Comment (anton-bartoshevich) [reply Anton_Bartoshevich/08.Docker.Docker-compose/docker-image.yml:5]: OK, I'll repare it

**Review by pluhin — APPROVED** [review]
Good, approved

**Pull request merged**